RS/RS/Service/GamePathService.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Microsoft.Win32;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace RS.Service
+{
+    internal class GamePathService
+    {
+        public GamePathService() { }
+
+        /// <summary>
+        /// Validate if the input game path is valid by finding the perferences.xml
+        /// </summary>
+        /// <param name="gamePath">The game path from the TextBox</param>
+        /// <returns>A boolean vakue indicating whther the path is valid</returns>
+        public bool ValidatePath(string gamePath)
+        {
+            // Check if we have a game path
+            Console.WriteLine("-> " + gamePath);
+            // Only check for perferences.xml (it should work for both steam and normal version)
+            return File.Exists(gamePath + @"\preferences.xml");
+        }
+
+        /// <summary>
+        /// Enable the replay feature of the game. 
+        /// Call <seealso cref="ValidatePath(string)"/> first to valid the path.
+        /// </summary>
+        /// <param name="gamePath">The game path from the TextBox</param>
+        public void EnableReplay(string gamePath)
```

Comment:
Not sure if this is still needed. Replay should be enabled by default now in 2023.